packages/transaction-controller/src/TransactionController.ts

```diff
@@ -208,10 +210,10 @@ export class TransactionController extends BaseController<
 
   private readonly pendingTransactionTracker: PendingTransactionTracker;
 
-  private failTransaction(transactionMeta: TransactionMeta, error: Error) {
+  private failTransaction(transactionMeta: TransactionMeta, error: TxError) {
```

Comment:
As we can fail a transaction for any error, and only normalize here, should this type remain as `Error` for clarity?